Editor/Halodi/PackageRegistry/Core/UpgradePackagesManager.cs

```diff
@@ -82,22 +82,22 @@ public PackageUpgradeState(UnityEditor.PackageManager.PackageInfo info)
                         }
                     }
                     
-                    hasVerified = !String.IsNullOrWhiteSpace(info.versions.verified);
+                    hasVerified = !String.IsNullOrWhiteSpace(info.versions.recommended);
                     if(hasVerified)
                     {
                         try
                         {
-                            verifiedVersion = SemVer.Parse(info.versions.verified);
+                            verifiedVersion = SemVer.Parse(info.versions.recommended);
```

Comment:
Breaks Unity 2021/2022.

Reply:
Added precompiled check to avoid issues with previous versions of Unity